src/lib/journey/stages/_utilities/step.utilities.test.ts

```diff
@@ -38,6 +42,26 @@ describe('Test string to key conversion', () => {
   });
 });
 
+describe('Test check validation', () => {
+  it('should return true with failed policies', () => {
+    const checkValidation = initCheckValidation();
+    const result = checkValidation(
```

Comment:
If there's a better / more correct way to call the function returned by `initCheckValidation()` I'm open to suggestions.

Reply:
Whatever way we can test our code, I'm good with it.